apps/arweave/src/ar_bench_packing.erl

```diff
@@ -13,7 +13,9 @@ run_benchmark_from_cli(Args) ->
 	LargePages = list_to_integer(get_flag_value(Args, "large_pages", "1")),
 	HardwareAES = list_to_integer(get_flag_value(Args, "hw_aes", "1")),
 	VDF = list_to_integer(get_flag_value(Args, "vdf", "0")),
-	run_benchmark(Test, JIT, LargePages, HardwareAES, VDF).
+	Iterations = list_to_integer(get_flag_value(Args, "iterations", "1")),
+	Rounds = list_to_integer(get_flag_value(Args, "rounds", "8")),
```

Comment:
Should the default rounds be 10? 

Or actually maybe [it is 8](https://github.com/ArweaveTeam/arweave/pull/590/files#r1706283377)? I think I'm missing something

Reply:
We need to fix it here.

Reply:
Okay great. I'll look at cleaning up the benchmarks as part of re-running the re-encrypt tests.